src/app/helpers/environmentVariablesHelper.js

```diff
@@ -11,8 +11,7 @@ let envVariables = {
   DEFAULT_CHANNEL: env.sunbird_default_channel,
 
   // discussion forum 
-  discussion_forum_token: env.discussion_forum_token || 'a4838b88-6a04-4293-a504-245862cad404',
-  discussions_middleware: env.discussions_middleware || 'http://disussionsmw-service:3002/discussion',
+  discussions_middleware: env.discussions_middleware || 'http://discussionsmw-service:3002',
```

Comment:
Please do not modify this file unless n until you're not adding any new variable into Portal...
Please revert this file changes

Reply:
these lines are same as release-3.6.0 branch